test/common.py

```diff
@@ -156,6 +141,10 @@ def tmp_digits_dataset(prefix: str,
                        sort_target: bool = False,
                        seed_train: int = 13, seed_dev: int = 13,
                        with_source_factors: bool = False):
+    """
```

Comment:
We could maybe even add a `Dict` return type.

Reply:
done